src/pyenphase/envoy.py

```diff
@@ -226,6 +228,27 @@ async def probe(self) -> None:
         else:
             self._supported_features |= SupportedFeatures.INVERTERS
 
+        # Check for various Ensemble support
+        if self._firmware.version >= ENSEMBLE_MIN_VERSION:
+            # The Ensemble Inventory endpoint will tell us if we have Enpower or Encharge support
+            try:
+                result = await self.request(URL_ENSEMBLE_INVENTORY)
+            except (json.JSONDecodeError, httpx.HTTPError) as e:
```

Comment:
We should probably make this a constant at some point

Reply:
What part? I think Github highlighted the wrong section

Reply:
(json.JSONDecodeError, httpx.HTTPError)